CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -38,7 +38,7 @@ namespace l1t {
       std::vector<unsigned> uparams_;
       std::vector<int> iparams_;
       std::vector<std::string> sparams_;
-      Node(){ type_="unspecified"; version_=0; }
+      Node(){ type_="unspecified"; version_=1; }
```

Comment:
Indeed thats our model.  We'll always support reading obsolete versions, but will never write anything but the latest version.